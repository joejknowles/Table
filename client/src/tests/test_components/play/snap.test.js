import Snap from '../../../components/play/snap';

import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';
const shallowRenderer = ReactTestUtils.createRenderer();

it('Snap renders as before', () => {
  const component = shallowRenderer.render(<Snap />);
  expect(component).toMatchSnapshot();
});
