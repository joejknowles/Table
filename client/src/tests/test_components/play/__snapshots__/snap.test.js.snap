exports[`test Snap renders as before 1`] = `
<Connect(Component)
  action={
    Object {
      "type": "SNAP",
    }
  }
  className="btn snap"
  res="Snap!" />
`;
